src/SW_Weather.c

```diff
@@ -882,6 +885,7 @@ void generateMissingWeather(
       "generateMissingWeather(): method = %u is not implemented.\n",

```

Comment:
`generateMissingWeather()` should return early on error after `SW_MKV_today()` -- since this is called in a loop